src/main/java/bio/terra/common/iam/SamUserAuthenticatedRequest.java

```diff
@@ -0,0 +1,120 @@
+package bio.terra.common.iam;
+
+import bio.terra.common.exception.UnauthorizedException;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.util.Objects;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+/** Class representing the identity of an authenticated user. */
+@JsonDeserialize(builder = SamUserAuthenticatedRequest.Builder.class)
+public class SamUserAuthenticatedRequest {
```

Comment:
I was mainly trying to stick with the preexisting terminology. I agree the names are clunky.

Reply:
I thought `request` was there because the information for creating the object came from the HTTP request object. I would not object to different names, but the ones you have are fine with me.

It does look to me like you have mismatched names between the factory and the object.